parasol_cpu/tests/test_src/args/args.c

```diff
@@ -0,0 +1,26 @@
+typedef struct Foo {
+    char a;
+    short b;
+    int c;
+    long long d;
+} Foo;
+
+
+void fn1(char a, short b, int c, long long d, long long *output) {
+    *output = a + b + c + d;
+}
+
+int fn2(char a, short b, int c, long long d) {
+    return a + b + c + d;
+}
+
+
+int fn3(long long a, int b, short c, char d) {
```

Comment:
Any chance we could define 128 bits?

Reply:
I'll need to change clang, but it's an easy fix. Can do in another check int.